client/transactions/list/index.js

```diff
@@ -149,14 +149,15 @@ export const TransactionsList = ( props ) => {
 		isLoading: isSummaryLoading,
 	} = useTransactionsSummary( getQuery(), props.depositId );
 
-	const columnsArgs = [
-		! props.depositId,
-		wcpaySettings.isSubscriptionsActive,
-		! getQuery().orderby || 'date' === getQuery().orderby,
-	];
+	const sortByDate = ! getQuery().orderby || 'date' === getQuery().orderby;
 	const columnsToDisplay = useMemo(
-		() => getColumns( ...columnsArgs ),
```

Comment:
Feeling confident in this change, since it would have been picked up by the snapshot tests.